src/helpers/uniqueAssetTraitDisplayTypeCompareFunction.ts

```diff
@@ -0,0 +1,29 @@
+import { UniqueAsset } from '@rainbow-me/entities';
+
+type Trait = UniqueAsset['traits'][number];
+
+// lower number = higher precedence
+const displayTypeRanks: Record<string, number> = {
```

Comment:
```suggestion
const displayTypeRanks = {
```

This typing is redundant with `const`. AFAIK

Reply:
Actually it is useful, we need to loosen the typing here, `as const` makes it so that I can only pick `boost_number | boost_percentage | date` and I really want the key to be a string since that is what opensea returns. That combined with nullish coalescing is pretty readable and works for Typescript